shared/productTypes.ts

```diff
@@ -445,6 +447,46 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
 			productFilenamePart: 'Newspaper - Home Delivery',
 		},
 	},
+	nationaldelivery: {
+		productTitle: calculateProductTitle('Newspaper Delivery'),
+		friendlyName: () => 'newspaper national delivery subscription',
```

Comment:
on the checkout I don't think we differentiated the national delivery with a different name, so maybe the same logic applies here?